cetmix_tower_server/models/cx_tower_variable_value.py

```diff
@@ -57,6 +57,8 @@ class TowerVariableValue(models.Model):
         compute="_compute_variable_ids",
         store=True,
     )
+    required = fields.Boolean(default=False)
```

Comment:
No need to set `False` explicitly because this is default value. Don't let AI fool you!))